packages/itwinui-css/src/table/base.scss

```diff
@@ -371,8 +371,7 @@
   > .iui-table-empty {
     text-align: center;
     position: sticky;
```

Comment:
Since we're using margin and not `position: sticky` + `inset-inline-start: 50%`, is `position: sticky` needed any more?

Reply:
I've removed `position: sticky`.